templates/_badgemenu.html

```diff
@@ -11,7 +11,7 @@
 					<a class="nav-btn" href="{{ genRoute "user_notifications" }}">{{ call $.T "my_notifications"}} <span class="badge">({{ .GetUnreadNotifications }})</span></a>
 					<a class="nav-btn" href="{{ genRoute "user_profile_edit" "id" (print .ID) "username" .Username }}">{{call $.T "settings"}}</a>
 					{{if HasAdmin . }}<a class="nav-btn" href="{{ genRoute "mod_index" }}">{{call $.T "moderation"}}</a>{{end}}
-					<a class="nav-btn" href="{{ genRoute "user_logout" }}">{{ call $.T "sign_out"}}</a></li>
+					<a class="nav-btn" href="{{ genRoute "user_logout" }}?token={{ $.LogoutToken }}">{{ call $.T "sign_out"}}</a></li>
```

Comment:
If tokens from gorilla don't expire (I haven't looked at that project) that could work too. I'm mostly concerned about it throwing errors or inconveniencing people that leave a page open for a while.